src/notification.c

```diff
@@ -237,6 +237,139 @@ check_value_type (const char *key,
   return FALSE;
 }
 
+static void
+markup_parser_text (GMarkupParseContext  *context,
+                    const gchar          *text,
+                    gsize                 text_len,
+                    gpointer              user_data,
+                    GError              **error)
+{
+  GString *composed = user_data;
+  gchar *str = (gchar *) text;
+
+  /* Convert new lines to spaces without adding multiple consecutive spaces */
+  while (*str)
+    {
+      gunichar c = g_utf8_get_char (str);
```

Comment:
This one needs the string to be valid utf8 for example...

Reply:
this also comes from a valid utf8 string then :)